README.md

```diff
@@ -350,45 +350,27 @@ Destroying models may result in errors due to foreign key integrity. First delet
 
 ## Running tests
 
-The tests in this repository are mainly integration tests, meaning you will need to run them using our preconfigured test server.
-
-1. Ask a core developer for instructions on how to set up test server
-   credentials on your machine
-2. `npm test`
-
-If you wish to run the tests using your own test database instance,
-
-__Set up the database__
-
-1. Go to pgAdmin.  
-By default, the local database is one of the servers under Server Groups > Servers.  
-2. Under Login Roles, add a user called ```strongloop```.
-
-__Change configuration for database connection__
+## Running tests
```

Comment:
is there any reason we remove the info about how to run the tests locally *without* docker?